tests/transformation_tests.py

```diff
@@ -0,0 +1,91 @@
+'''
+Copyright 2015 Planet Labs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+import unittest
+import numpy
+
+from radiometric_normalization import transformation
+
+
+class Tests(unittest.TestCase):
+    def test_get_transform_function(self):
+        self.assertEqual(
+            transformation.get_transform_function('linear_relationship'),
+            transformation.linear_relationship)
+        self.assertRaises(Exception,
+                          transformation.get_transform_function,
+                          'unknown')
+
+    def test_pifs_to_pifset(self):
```

Comment:
lol I find the name "pifset" really confusing because I'd expect it to just be a [set](https://jobs.planet-labs.com/v0/jobs/25731632) of pifs!